doc/acl/acl_stage_capability.md

```diff
@@ -86,6 +106,7 @@ private:
    void queryAclCapabilities();
 
    std::map<sai_acl_stage_t, std::set<sai_acl_action_type_t>> m_aclStageCapabilities;
+   std::map<sai_acl_entry_attr_t, std::set<int32_t>> m_aclEnumActionCapabilities;
```

Comment:
change to m_aclPacketActionCapabilities?

Reply:
@stcheng This map is not only for PACKET_ACTION, there is also ACTION_DTEL_FLOW_OP which value is enum and can be validated